docs/_sources/tutorials/blue-button-20.rst.txt

```diff
@@ -67,12 +67,16 @@ The result indicates the topic, partition and offset of the message in Kafka.  '
 
 View the NATS Notification (Optional)
 -------------------------------------
-You should also see a NATS notification in the nats-subscriber service log.  The message received by the NATS subscriber also indicates the topic, partition and offset of the message in Kafka, which could be used for downstream application integration.
+You should also see a NATS notification in the Linux for Health connect log.  If you are running the Linux for Health connect application locally, you should see a message in the connect console window similar to::
 
-To view NATS notifications in a new console window::
+   15:14:29.474 [nats:3] INFO  c.l.connect.support.NATSSubscriber - nats-subscriber-localhost:4222-lfh-events received message: {"meta":{"routeId":"fhir-r4-rest","uuid":"8bebaaae-a30b-4d8e-8424-d38836bf1d14","routeUri":"jetty:http://0.0.0.0:8080/fhir/r4/Patient?httpMethodRestrict=POST","dataFormat":"FHIR-R4","messageType":"PATIENT","timestamp":1597868068,"dataStoreUri":"kafka:FHIR-R4_PATIENT?brokers=localhost:9092","status":"success","dataRecordLocation":["FHIR-R4_PATIENT-0@21"]}}
+
+If you are running Linux for Health connect with docker-compose, in a console window, navigate to the connect compose directory and view the logs::
```

Comment:
running a Linux for Health container within docker-compose,

Reply:
Made that text change and rebuilt the site based on the latest master, so ready to merge.